applications/integrationtests/mechanics/Feti.cpp

```diff
@@ -31,11 +33,16 @@ using NuTo::Interpolation::eTypeOrder;
 using NuTo::Interpolation::eShapeType;
 using NuTo::eGroupId;
```

Comment:
what is the point of these `using`s, if you have `using namespace NuTo` anyways?